src/file.toit

```diff
@@ -214,13 +245,17 @@ write_ descriptor data from to:
 close_ descriptor:
   #primitive.file.close
 
-// Delete a file, given its name.  Like 'rm' and the 'unlink()' system call,
-// this only removes one hard link to a file. The file may still exist if there
-// were other hard links.
-delete name:
+/**
+Deletes a file, given its name.
+Like 'rm' and the 'unlink()' system call, this only removes one hard link to a
+  file. The file may still exist if there were other hard links.
```

Comment:
I think that's too low level. Most people don't even know what hard links are.
Also, it's the expected behavior, so no need to put it here (given that it's a bit too low-level imho).